evm.md

```diff
@@ -1552,7 +1571,8 @@ For each `CALL*` operation, we make a corresponding call to `#call` and a state-
     syntax TernStackOp ::= "CREATE"
  // -------------------------------
     rule <k> CREATE VALUE MEMSTART MEMWIDTH
-          => #checkCall ACCT VALUE
+          => #touchAccounts #newAddr(ACCT, NONCE)
+          ~> #checkCall ACCT VALUE
```

Comment:
Are the accounts supposed to be touched _before_ the `#checkCall`? As in, if the call fails to go through (because not enough funds, for example), are we supposed to still touch the new account?

If it doesn't matter (which it should matter, but maybe the test-suite does not test that functionality), then we can have `#create` call `#touchAccounts` instead, which would only affect one rule, instead of both rules for `CREATE` and `CREATE2`.

Reply:
From EIP-2929 
> When a CREATE or CREATE2 opcode is called, immediately (ie. before checks are done to determine whether or not the address is unclaimed) add the address being created to accessed_addresses, but gas costs of CREATE and CREATE2 are unchanged. Clarification: If a CREATE/CREATE2 operation fails later on, e.g during the execution of initcode or has insufficient gas to store the code in the state, the address of the contract itself remains in access_addresses (but any additions made within the inner scope are reverted).

So, to answer your first two questions, yes.
Regarding this
> If it doesn't matter (which it should matter, but maybe the test-suite does not test that functionality), then we can have `#create` call `#touchAccounts` instead, which would only affect one rule, instead of both rules for `CREATE` and `CREATE2`.

I could also add `#gasAccess` rules for both of `CREATE` and `CREATE2` and  deduct 0 gas. 
` #gasAccess(_    , CREATE _ _ _)   => #touchAccounts #newAddr(ACCT, NONCE) ~> 0`

Reply:
From EIP-2929 
> When a CREATE or CREATE2 opcode is called, immediately (ie. before checks are done to determine whether or not the address is unclaimed) add the address being created to accessed_addresses, but gas costs of CREATE and CREATE2 are unchanged. Clarification: If a CREATE/CREATE2 operation fails later on, e.g during the execution of initcode or has insufficient gas to store the code in the state, the address of the contract itself remains in access_addresses (but any additions made within the inner scope are reverted).

So, to answer your first two questions, yes.
Regarding this
> If it doesn't matter (which it should matter, but maybe the test-suite does not test that functionality), then we can have `#create` call `#touchAccounts` instead, which would only affect one rule, instead of both rules for `CREATE` and `CREATE2`.

I could also add `#gasAccess` rules for both of `CREATE` and `CREATE2` and  deduct 0 gas. 
` #gasAccess(_    , CREATE _ _ _)   => #touchAccounts #newAddr(ACCT, NONCE) ~> 0`

Reply:
No worries, thanks for answering my question!